.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -13,7 +13,9 @@ body:
   - type: input
     attributes:
       label: Version
-      description: Output of `node -v`
+      description: |
+        Output of `node -v`.
+        Please verify that you are reproducing the issue in a currently-supported version of Node.js.
```

Comment:
Can you add a link to https://github.com/nodejs/Release?tab=readme-ov-file#release-schedule?